README.md

```diff
@@ -1 +1,36 @@
-# Portfolio-setup-and-mobile-version-skeleton
\ No newline at end of file
+# Portfolio-setup-and-mobile-version-skeleton
+
+> This is the first milestone in the process of creating my portfolio website.
+
+![screenshot](Images/Screenshot.png)
+
+This website is a setup and mobile version skeleton of my portfolio.
+
+## Built With
+
+- HTML
+- CSS
+- VSCode
+
```

Comment:
- [OPTIONAL] Let's imagine someone interested in your project and wants to clone and use it locally, but there are no instructions on how to set up a project locally section in your Readme. Kindly add it above the Authors section. 😄

Reply:
Thank you for the information.